CHANGELOG.rst

```diff
@@ -2,6 +2,10 @@
 Changelog
 =========
 
+3.3.9 (2022-08-16)
+* [MOD] Mapper: Replace missing column parameters (`nullify_missing_columns`, `skip_missing_columns`,
+`ignore_missing_columns`) with one single parameter: `missing_column_handling`
```

Comment:
Probably good to mention that the default behavior changed from nullifying missing columns to raise an error

Reply:
The default behaviour did not really change, it also raised an error in the previous versions

Reply:
Sorry, my bad. I thought the default was still to nullify missing columns.